backend/util/tokenDecoder.js

```diff
@@ -0,0 +1,11 @@
+const atob = require('atob');
+
+//decodes given token
+function tokenDecoder(req) {
+	const authHeader = req.headers['authorization']
+	const token = authHeader.split(' ')[1]
+	const decodedToken = atob(token.split('.')[1])
+	return JSON.parse(decodedToken)
+}
+
+module.exports = {tokenDecoder}
```

Comment:
Please consider using jwt.verify(), you can pull token data from there easily.

Reply:
Okay, i deleted this function so that we can combine verifying and decoding the token